### ISSUE TITLE
Is the new Filter list (x/y) indicator a good thing?

### ISSUE THREAD

**Issue by joey04** [uBlockOrigin/uBlock-issues#27]
### Prerequisites

- [ ] I verified that this is not a filter issue
    - Filter issues MUST be reported at [filter issue tracker](https://github.com/uBlockOrigin/uAssets/issues)
    
- [ ] This is not a support issue or a question
    - Support issues and questions are handled at [/r/uBlockOrigin](https://www.reddit.com/r/uBlockOrigin/)
    
- [ ] I performed a cursory search of the issue tracker to avoid opening a duplicate issue
    - Your issue may already be reported.
- I tried to reproduce the issue when...
    - [ ] uBlock Origin is the only extension
    - [ ] uBlock Origin with default lists/settings
    - [ ] using a new, unmodified browser profile
- [ ] I am running the latest version of uBlock Origin
- [ ] I checked the [documentation](https://github.com/gorhill/uBlock/wiki) to understand that the issue I report is not a normal behavior

### Description

I like the recent enhancements to the look of the Filter list tab of the dashboard, but I question if the (x/y) indicator for each list section is a good thing. See my screenshot below for an example.

I suspect it will encourage users, namely the many not-so-informed ones, to think "Oh, I don't have all of them selected, maybe I should." I recall seeing a few screenshots of this on the main tracker, including complete disregard for obvious redundancies. (So it's not something that is preventable, of course, just shouldn't be subtly suggested.)

gorhill, you've expressed concern about not overburdening list servers (including GitHub) so that's why it's worth mentioning here. 

![untitled](https://user-images.githubusercontent.com/14959062/39491457-02bd055a-4d7c-11e8-99b9-0472385b0ee7.png)
### Your environment

* uBlock Origin version: 1.16.3b1
* Browser Name and version: Firefox 59

**Comment by gwarser** [comment]
You proposition? Leave only `+`?

**Comment by uBlock-user** [comment]
> I suspect it will encourage users, namely the many not-so-informed ones, to think "Oh, I don't have all of them selected, maybe I should."

There hasn't been the case where the users are discouraged from subscribing to more lists. If they want to, they can.

**Comment by joey04** [comment]
I think it'd be better to restore the way it was.

Here's 1.14.16:

![untitled](https://user-images.githubusercontent.com/14959062/39492746-93166c22-4d5e-11e8-88a2-d2a6a253e040.png)

**Comment by gwarser** [comment]
Some indication, that we have more filter lists available, is needed. In your screenshot only one `+` in first line suggests that. Numbers in brackets duplicate what you already see.

**Comment by joey04** [comment]
> Some indication, that we have more filter lists available, is needed.

Agreed. I'm only talking about (x/y) here. Everything else is an improvement, including the new "Built-in" category for uAssets lists.

I'm only suggesting remove the (x/y) and just have the (x) for categories with one or more lists selected, like it was in 1.14.16

**Comment by joey04** [comment]
By "some indication" I thought he was referring to the `+` next to each category name in the first screenshot. That's what I was agreeing with.

So I think that's indication enough that there are more lists available. Hence either just the `(x)` like before (which he pointed out is redundant and I agree) or remove it altogether.
